vnext/ReactWindowsCore/WinRTWebSocketResource.h

```diff
@@ -59,6 +59,8 @@ class WinRTWebSocketResource : public IWebSocketResource, public std::enable_sha
       winrt::Windows::Networking::Sockets::IMessageWebSocketMessageReceivedEventArgs const &args);
   void Synchronize();
 
+  std::string GetRestrictedErrorMessage() noexcept;
```

Comment:
> std::string [](start = 1, length = 12)

[nit] Assuming 'GetRestrictedErrorMessage()' represents state that does not necessarily change between invocations, would it be more performant to cache the error message and return a std::string_view or a 'const char*'?

Reply:
Don't think so. We would want to query for it on any error condition, on which such message may have changed (connect, read or write error messages differ).